index.html

```diff
@@ -1348,15 +1342,15 @@ <h2>Successfully relocate a node from source to destination section</h2>
                 Contains:<br />
                 <ul>
                     <li>target_interval: (XorName, XorName) - The interval into which the joining node should join.<br>
-                    <li>section_info: SectionInfo - The destination section that the joining node will trust.<br>
+                    <li>section_info: SectionInfo - The destination section that the joining node will trust and connect too.<br>
                     <li>proof: Signatures - Quorum of signature for the candidate to prove the source section relocated
                         it with these infos.<br>
                 </ul>
             </p>
         </div>
         <button class="collapsible">CandidateInfo</button>
         <div class="content">
-            <p>Sent by the joining node to the target interval's NaeManager to initiate the joining process<br />
+            <p>Sent by the joining node to each elders of the section it is to initiate the joining process<br />
```

Comment:
s/elders/elder/

I don't get "the section it is". Are you missing the word: "joining"?